CONTRIBUTORS

```diff
@@ -1,9 +1,12 @@
 Maintainers:
 
 	Base:    Tulio Magno Quites Machado Filho <tuliom@linux.vnet.ibm.com>
+	         Paul E. Murphy <murphyp@linux.vnet.ibm.com>
 	PowerPC: Tulio Magno Quites Machado Filho <tuliom@linux.vnet.ibm.com>
-	s390:    <tbd>
+	         Paul E. Murphy <murphyp@linux.vnet.ibm.com>
+	s390:    Paul E. Murphy <murphyp@linux.vnet.ibm.com>
 	x86(64): Tulio Magno Quites Machado Filho <tuliom@linux.vnet.ibm.com>
+	         Paul E. Murphy <murphyp@linux.vnet.ibm.com>
```

Comment:
What about putting your name before mine?

Does it make sense to distinguish all of this nowadays?  I wonder if just a list of maintainers would be enough.